subsystem/cluster_v2_test.go

```diff
@@ -488,7 +489,7 @@ var _ = Describe("[V2ClusterTests] multiarch", func() {
 		})
 
 		Expect(err).To(HaveOccurred())
-		actual := err.(*installer.RegisterInfraEnvBadRequest)
-		Expect(*actual.Payload.Reason).To(ContainSubstring("does not have a matching OpenShift release image"))
+		actual := err.(*runtime.APIError)
```

Comment:
Ah I guess this is failing because you added the enum, right?

Can you also just change the name of the test?
Maybe something like `Fails to register an infraenv with a non-supported OS architecture` or something to distinguish it from the release/OS images provided in the config since that's actually not relevant to this test anymore.